### PR TITLE
Add step to patch 4.x default storage class to non-default

### PR DESCRIPTION
**Pull request by liangxia**
@akostadinov @pruan-rht @chao007

### CODE REVIEW

**Comment by liangxia** [comment]
Any updated ?

**Comment by liangxia** [comment]
@jhou1 @qinpingli   Could you help review ?

**Comment by liangxia** [comment]
Actually this one is quite urgent since using the old way (3.x) to delete default storage class in 4.x will cause auto execution quit prematurely

**Comment by qinpingli** [comment]
LGTM

File: features/step_definitions/storage_class.rb [hunk 1]
```diff
@@ -77,6 +77,29 @@
   @result = host.exec_admin(*table.raw.flatten)
 end
 
+Given(/^default storage class is patched to non-default$/) do
+  ensure_admin_tagged
+  ensure_destructive_tagged
+
+  _sc = BushSlicer::StorageClass.get_matching(user: user) { |sc, sc_hash| sc.default? }.first
+  if _sc
+    logger.info "Default storage class will patched to non-default and be resotored after scenario:\n#{_sc.name}"
```
Comment (akostadinov) [inline features/step_definitions/storage_class.rb]: Here you could add
```
    cache_resources _sc
```

Comment (akostadinov) [reply features/step_definitions/storage_class.rb]: otherwise lgtm

**Pull request merged**